### PR TITLE
IGNITE-11772: [ML] Broken javadoc

### PR DESCRIPTION
**Pull request by ybabak**
fixed javadocs

### CODE REVIEW

File: modules/ml/src/main/java/org/apache/ignite/ml/composition/boosting/convergence/simple/ConvergenceCheckerStub.java [hunk 1]
```diff
@@ -46,7 +46,6 @@
      * @param externalLbToInternalMapping External label to internal mapping.
      * @param loss Loss function.
      * @param datasetBuilder Dataset builder.
```
Comment (dmitrievanthony) [inline modules/ml/src/main/java/org/apache/ignite/ml/composition/boosting/convergence/simple/ConvergenceCheckerStub.java]: I see `double precision` and `preprocessor` parameter which is not mentioned in javadoc.

File: modules/ml/src/main/java/org/apache/ignite/ml/genetic/MutateTask.java [hunk 2]
```diff
@@ -78,7 +78,7 @@ public MutateTask(GAConfiguration cfg) {
      * @param nodes List of ClusterNode
      * @param chromosomeKeys Primary keys for respective chromosomes
```
Comment (dmitrievanthony) [inline modules/ml/src/main/java/org/apache/ignite/ml/genetic/MutateTask.java]: Where is the return statement in javadoc?

File: modules/ml/src/main/java/org/apache/ignite/ml/composition/boosting/convergence/simple/ConvergenceCheckerStub.java (hunk 1 above)
Comment (ybabak) [reply modules/ml/src/main/java/org/apache/ignite/ml/composition/boosting/convergence/simple/ConvergenceCheckerStub.java]: fixed

File: modules/ml/src/main/java/org/apache/ignite/ml/genetic/MutateTask.java (hunk 2 above)
Comment (ybabak) [reply modules/ml/src/main/java/org/apache/ignite/ml/genetic/MutateTask.java]: fixed

**Pull request closed without merging**